### DISCUSSION TITLE
Additional lora (more than 5)

### DISCUSSION THREAD

**Discussion by kuduate** [lllyasviel/Fooocus#1268] [Q&A] [answered]
Hi,

Is there any possibility to add additional loras? By the default Fooocus seems to support up to five. I tried to add it via the config file, but it is not loaded. 

`Request to load LoRAs [['1.safetensors', 0.59], ['2.safetensors', 0.59], ['3.safetensors', 0.9], ['4.safetensors', 0.66], ['5.safetensors', 0.61]] for model [G:\Fooocus AI\Fooocus\models\checkpoints\XL_realcartoon_v5.safetensors].`

`	"default_loras": [
	[
		"1.safetensors",
		0.59
	],
	[
		"2.safetensors",
		0.59
	],
	[
		"3.safetensors",
		0.9
	],
	[
		"4.safetensors",
		0.66
	],
	[
		"5.safetensors",
		0.61
	],
	[
		"6.safetensors",
		0.75
	]
	],`

**Comment by w0lfram** [comment]
I'll second that. Would really like to be able to use more than 5 loras. 10 would be just fine.

**Comment by CastomStrange** [comment, marked as answer]
Add it to the file config.txt this line  "default_max_lora_number": 10

**Reply by UK-Slinger** [reply]
Many thanks.
